app.js

```diff
@@ -1 +1,43 @@
-console.log("js file is connected")
\ No newline at end of file
+const rootUrl = "https://api.edamam.com/search";
+const recipeUrlElement = document.querySelector("#recipe-url");
+const labelElement = document.querySelector("#label");
+const imageElement = document.querySelector("#cake-img");
+let ingredientsListElement = document.querySelector(".ingredients");
+
+const apiUrl = `${rootUrl}?q=birthday+cake+strawberry&app_id=589ecbd6&app_key=6d6116bfcbdc60fe641222727dc9eb8f`;
+
+const form = document.querySelector("#choose-ingredient-form");
+// list of all input elements
+const inputElements = document.querySelectorAll("input[name='ingredient']");
+form.addEventListener("submit", searchIngredient);
+
+fetchRecipies(apiUrl);
```

Comment:
I would just run `searchIngredient` here instead of `fetchRecipies`, then you don't need to define `apiUrl` variable manually and could get rid of it.

Reply:
good one